utils/vscode/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "carbon-lang",
+  "displayName": "carbon-lang",
+  "description": "Carbon Language support",
+  "version": "0.0.1",
+  "repository": {
+    "url": "https://github.com/carbon-language/carbon-lang"
+  },
+  "engines": {
+    "vscode": "^1.70.1"
```

Comment:
I only tested 1.70.1 and 1.79.2
Likely older versions are also supported but this is old enough (August 2022)